src/Microsoft.DotNet.Web.ProjectTemplates/content/RazorPagesWeb-CSharp/wwwroot/lib/bootstrap/.bower.json

```diff
@@ -29,7 +29,7 @@
     "test-infra"
   ],
   "dependencies": {
-    "jquery": "1.9.1 - 3"
+    "jquery": "3.3.1"
```

Comment:
Where does this change come from? Was it authored by hand, or is this the content from an updated version of the `bootstrap` package from Bower?